qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/fitting/plotting/test/test_plot_model.py

```diff
@@ -120,13 +120,15 @@ def _setup_update_fit_test(self, mock_table_values, mock_ads, mock_get_diffs, fu
         }
         return fitprop
 
+    @patch(plot_model_path + ".FittingPlotModel._ws_is_tof")
     @patch(plot_model_path + ".FittingPlotModel._get_diff_constants")
     @patch(plot_model_path + ".FittingPlotModel.create_fit_tables")
     @patch(plot_model_path + ".ADS")
-    def test_update_fit(self, mock_ads, mock_create_fit_tables, mock_get_diffs):
+    def test_update_fit(self, mock_ads, mock_create_fit_tables, mock_get_diffs, mock_ws_is_tof):
         mock_loaded_ws_list = mock.MagicMock()
         mock_active_ws_list = mock.MagicMock()
         mock_log_ws_name = mock.MagicMock()
+        mock_ws_is_tof.return_value = True
```

Comment:
Is it worth adding a test to make sure if `mock_ws_is_tof.return_value = False` the table `Gaussian_PeakCentre_dSpacing` doesn't exist?